shared/src/helpers/user/get-avatar-initials.ts

```diff
@@ -1,9 +1,12 @@
 const getAvatarInitials = (fullName: string): string => {
     const [name, secondName] = fullName.split(' ');
     const firstCharIndex = 0;
-    return `${name[firstCharIndex].toUpperCase()}${secondName[
-        firstCharIndex
-    ].toUpperCase()}`;
+
+    return secondName
```

Comment:
we should use or name or email. Because we can sign up and after we will have only email

Reply:
we can pass any string there and we will get first letter is it's email.